source/adapters/hip/memory.cpp

```diff
@@ -274,7 +274,12 @@ UR_APIEXPORT ur_result_t UR_APICALL urMemGetInfo(ur_mem_handle_t hMemory,
           return AllocSize;
         } else if constexpr (std::is_same_v<T, SurfaceMem>) {
           HIP_ARRAY3D_DESCRIPTOR ArrayDescriptor;
-          UR_CHECK_ERROR(hipArray3DGetDescriptor(&ArrayDescriptor, Mem.Array));
+#if HIP_VERSION_MAJOR >= 5 && HIP_VERSION_MINOR >= 6
```

Comment:
If I get it right, with this patch the feature will remain unavailable in 6.0. Isn't it better to use [HIP_VERSION](https://rocmdocs.amd.com/projects/HIP/en/develop/user_guide/faq.html#how-can-i-know-the-version-of-hip) macro instead?

Reply:
Yes you are correct. Have changed to `HIP_VERSION`